README.md

```diff
@@ -83,8 +83,16 @@ optionally `npm run clean` to clean out any `node_modules` folders installed by
 
 # How to contribute?
```

Comment:
Not sure if this is the right section, but here are a few additional things, that maintainer(s) could do to reinforce standards and code discipline as part of the responsibilities
- Ensure branch naming conventions: some good references here, https://gist.github.com/digitaljhelms/4287848
- Ensure linting conventions are followed for committing changes
- Ensure PR naming conventions
- Ensure end to end CI process is built in
Feel free to add any other